README.md

```diff
@@ -31,6 +31,21 @@ Download the most recent `Monocraft.otf` file from the [Releases](https://github
 
 Download the most recent `Monocraft.otf` file from the [Releases](https://github.com/IdreesInc/Minecraft-Mono/releases) page. Double click on the downloaded font file and select **Install Font** in the window that appears. More help available [here](https://support.apple.com/en-us/HT201749).
 
+### Linux
+
+#### Arch
+
+Download Monocraft.otf, cp to /etc/fonts:
+```bash
+curl -O https://github.com/IdreesInc/Monocraft/releases/download/v1.2/Monocraft.otf
```

Comment:
It may be better to link to the most recent release, to not have to deal with updating in future revisions.
E.g. like so: `https://github.com/IdreesInc/Monocraft/releases/latest/download/Monocraft.otf`